apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/sql/SqlStore.java

```diff
@@ -0,0 +1,92 @@
+package com.apollographql.android.cache.normalized.sql;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteStatement;
+
+import com.apollographql.android.cache.normalized.CacheStore;
+import com.apollographql.android.cache.normalized.Record;
+
+import java.sql.SQLException;
+
+import javax.annotation.Nullable;
+
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.COLUMN_KEY;
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.COLUMN_RECORD;
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.TABLE_RECORDS;
+
+final class SqlStore extends CacheStore {
+
+  // Database fields
+  private SQLiteDatabase database;
+  private ApolloSqlHelper dbHelper;
+  private String[] allColumns = {ApolloSqlHelper.COLUMN_ID, ApolloSqlHelper.COLUMN_KEY, ApolloSqlHelper.COLUMN_RECORD};
+  private SQLiteStatement sqLiteStatement;
+  private String insertStatement;
+
+  public static SqlStore create(ApolloSqlHelper helper) {
+    return new SqlStore(helper);
+  }
+
+  private SqlStore(ApolloSqlHelper dbHelper) {
+    this.dbHelper = dbHelper;
+    insertStatement = String.format("INSERT INTO+%s(%s,%s) VALUES (?,?)",
+        TABLE_RECORDS,
+        COLUMN_KEY,
+        COLUMN_RECORD);
+  }
+
+
+  @Nullable @Override public Record loadRecord(String key) {
+    return null;
+  }
+
+  @Override public void merge(Record object) {
+
+  }
+
+
+  public Record createRecord(String key, String record) {
+    sqLiteStatement = database.compileStatement(insertStatement);
+    sqLiteStatement.bindString(0, key);
+    sqLiteStatement.bindString(0, record);
+
+    long recordId = sqLiteStatement.executeInsert();
```

Comment:
Always didn't understand why Sqlite doesn't return cursor right away to inserted row :(